core/modules/replica/WorkerProcessor.cc

```diff
@@ -46,34 +46,33 @@ namespace {
 
 LOG_LOGGER _log = LOG_GET("lsst.qserv.replica.WorkerProcessor");
 
-namespace replica = ::lsst::qserv::replica;
-namespace proto   = ::lsst::qserv::proto;
+using namespace lsst::qserv::replica;
 
 template <class PROTOCOL_RESPONSE_TYPE,
           class PROTOCOL_REQUEST_TYPE>
 bool ifDuplicateRequest(PROTOCOL_RESPONSE_TYPE& response,
-                        replica::WorkerRequest::Ptr const& p,
+                        WorkerRequest::Ptr const& p,
                         PROTOCOL_REQUEST_TYPE const& request) {
 
     bool isDuplicate = false;
 
-    if (replica::WorkerReplicationRequest::Ptr ptr =
-        dynamic_pointer_cast<replica::WorkerReplicationRequest>(p)) {
+    if (WorkerReplicationRequest::Ptr ptr =
```

Comment:
`auto` would work here and then you wouldn't have to line wrap which would be nice. 
implicit pointer conversion to bool is [disallowed](https://developer.lsst.io/cpp/style.html#implicit-test-for-0-should-not-be-used-other-than-for-boolean-variables)

Reply:
God points. Fixed.